### DISCUSSION TITLE
error while running roboshop-ansible

### DISCUSSION THREAD

**Discussion by LavanyaRamesh319** [learndevopsonline/learndevopsonline#1891] [Q&A]
I am getting error in roboshop-ansible after creating parameters. could you please help me to resolve this

workstation ip= 3.88.158.231
Jenkins ip= 54.242.128.4

**Comment by chaithanya1812** [comment]
Hi @LavanyaRamesh319 

## Expected values :-  reloaded, restarted, started, stopped,

![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/aba7dc0e-fe98-424d-bc0a-be4fa263c7be)

**Comment by LavanyaRamesh319** [comment]
Thank you @chaithanya1812. Its working fine now
